src/xcode/ENA/ENA.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
Did I miss the decision to delete the revolved file from git?

Reply:
this was not intended - rolled back the commit